crates/holochain_sqlite/src/schema/p2p/gossip_query.sql

```diff
@@ -0,0 +1,34 @@
+SELECT agent
+FROM p2p_store
+WHERE signed_at_ms >= :since_ms -- between given signed_at range
+  AND signed_at_ms <= :until_ms
+  AND (
+    -- if the input has two ranges, check them both
+    (
+      :storage_start_1 IS NOT NULL
+      AND :storage_end_1 IS NOT NULL
+      AND :storage_start_2 IS NOT NULL
+      AND :storage_end_2 IS NOT NULL
+      AND (
+        (
+          storage_center_loc >= :storage_start_1
+          AND storage_center_loc <= :storage_end_1
+        )
+        OR
+        (
+          storage_center_loc >= :storage_start_2
+          AND storage_center_loc <= :storage_end_2
+        )
+      )
+    )
+    OR
+    -- if the input has only one range, check it
+    (
+      :storage_start_1 IS NOT NULL
+      AND :storage_end_1 IS NOT NULL
+      AND storage_center_loc >= :storage_start_1
+      AND storage_center_loc <= :storage_end_1
+    )
+    -- if the input has no ranges, no records will match
+  )
+;
```

Comment:
would prefer SELECT-style sql files to be elsewhere, and keep the `schema/**` for schema definition and migrations

Reply:
@maackle - I'm accustomed to working with stored procedures in something like postgres, so i tend to think of all the sql as the schema... though I can see how that's not strictly true in the sqlite case...

I've moved the sql into a `sql` directory, and now instead of `initial.sql` it's `schema_initial.sql`, what do you think of that?

Reply:
@neonphog I was hoping to have a separate folder just for database migrations. I was thinking there'd be a directory with just:

- initial.sql
- 1.sql
- 2.sql

etc.

I'm indifferent as to what the overall folder is called but was hoping to keep migrations grouped (even though now there's only the initial one).

Reply:
Guess it could just `schema_initial.sql`, `schema_1.sql`, etc., but seems nicer in its own dir

Reply:
: ) I'm trying to avoid having them in different directories... If I change the schema, there's a very good chance everything else is going to have to change too - I want all the sql together, it's all interrelated.

Reply:
@neonphog OK that's fine. How do you think new migrations should be named?

Reply:
If I'm reading your plan for the migration code correctly, you'll actually have separate schema files for each version, as well as separate migration sql from previous version and potentially separate reverse migration sql...

so maybe something like:
- `schema_v0.sql`
- `schema_v1.sql`
- `schema_migrate_v0_to_v1.sql`
- `schema_migrate_v1_to_v0.sql`

Reply:
I'm not sure the reverse migration thing makes sense... we'll only have query/insert sql written for the current schema version... so I don't see how going backward would be useful

Reply:
updated